src/main/java/ee/tuleva/onboarding/user/exception/UserAlreadyAMemberException.java

```diff
@@ -0,0 +1,9 @@
+package ee.tuleva.onboarding.user.exception;
+
+public class UserAlreadyAMemberException extends RuntimeException {
```

Comment:
Nope, this is just a sanity check. Won't get thrown as of now (controller has an extra check for this).